find/search.py

```diff
@@ -267,7 +267,7 @@ def set_where(name: str) -> str:
     return '(c.id IN (SELECT card_id FROM printing WHERE set_id IN (SELECT id FROM `set` WHERE name = {name} OR code = {name})))'.format(name=sqlescape(name))
 
 def format_where(term: str) -> str:
-    if term == 'pd' or term == 'Penny Dreadful':
+    if term == 'pd' or term.startswith('penny'):
```

Comment:
Doesn't this cause issues with `f:"Penny Dreadful AER"`?

Reply:
Hm, yes, it does. I hadn't really realized or had forgotten that we supported that but I guess we do. I don't suppose anyone is using it though. We should introduce a shorter syntax for that. `pd16`/`penny16`? Dreadrise uses `pds16` so we could roll with that for compatibility.